Cargo.toml

```diff
@@ -19,3 +19,8 @@ serde = "0.9"
 serde_derive = "0.9"
 serde_json = "0.9"
 tokio-core = "0.1"
+rpassword = "0.4.2"
```

Comment:
Also, bump the `version` to `0.6.0`, and feel free to add yourself under the `authors` list :)